ampersand-state.js

```diff
@@ -756,11 +760,11 @@ var dataTypes = {
             // if this has changed we want to also handle
             // event propagation
             if (previousVal) {
-                this.stopListening(previousVal);
+                this.stopListening(previousVal, 'all', this._getCachedEventBubblingHandler(attributeName));
```

Comment:
Also, AFAIK the number of properties is fixed so this should never get out of hand.